indico/modules/events/papers/models/revisions.py

```diff
@@ -182,3 +182,22 @@ def get_reviewed_for_groups(self, user, include_reviewed=False):
         if self.paper.cfp.layout_reviewing_enabled and user in self.paper.cfp.layout_reviewers:
             reviewed_for.add(PaperReviewType.layout)
         return set(map(PaperTypeProxy, reviewed_for))
+
+    def has_user_reviewed(self, user, review_type=None):
```

Comment:
`review_type` should already be a PaperReviewType, not a string. So no need to convert it below